EIPS/eip-draft-signed-packages.md

```diff
@@ -0,0 +1,94 @@
+ - authors can become financially incentivized to develop and maintain(!) secure open source software (zero-config donation buttons, auto revenue splitting) which posed to be one of the biggest security problems of the past
+- it helps projects to manage donations and payments through a single software defined and package-contained source which makes project funding transparent, granular and makes it easier for projects to exist independent of parent organizations or donation platforms
+- it helps to transition away from centralized PKI and certificate authorities
+- it allows to move to privacy preserving trust models instead
+- we can issue and accept self-signed certificates with short expiration times and dedicated key usage which makes the certificate issuance, identity verification and signing process much faster and efficient and further limits the power of stolen certificates and keys
+- certificates can be defined to match and cover real world usage / identity proofs: e.g. "author has publishing access to git repository X" instead of "author has *common name* Y, works for organization W, and lives at address Z"
+- instead of having to pay hundreds of USD per year for a centrally issued and verified certificate, authors could get paid to sign their code and make modules more secure
+- leverages existing and emerging hardware such as "hardware wallets" to sign code -> no need to buy proprietary usb tokens and install drivers with lock-in effects
+- benefits not just the crypto ecosystem but the wider Web infrastructure
+- software licensing and auto-payments are possible which improves the portability of apps and packages and allows for decentralized software marketplaces 
+- helps to decouple development from distribution and hosting
+- compatible to emerging blockchain based identity provider solutions
+- leverages existing and emerging ecosystem tools for key management such as keybase or metamask to manage keys and sign packages in the browser
+- certificate revocation lists can be implemented on the blockchain to be tamper proof and "instantly" updated to prevent bad actors from using stolen keys to sign malware
+
+Discussion on the magicians forum: [Package and Dependency Managers](https://ethereum-magicians.org/t/package-and-dependency-managers/2418)
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+A specification draft and reference implementation can be found at:
```

Comment:
I think the key point of ERCs is to include actual specification. Would it be possible to include an excerpt here at least or to point to a specification commit hash?